en/frames_rover/index.md

```diff
@@ -19,7 +19,7 @@ PX4 supports rovers with:
 
 - **Differential steering**: direction is controlled by moving the left- and right-side wheels at different speeds.
   This kind of steering commonly used on bulldozers, tanks, and other tracked vehicles.
-- **Ackermann steering**: direction is controlled by pointing wheels in the direction of travel ([ackermann geometry](https://en.wikipedia.org/wiki/Ackermann_steering_geometry) compensates for the fact that wheels on the inside and outside of the turn move at different rates).
+- [**Ackermann steering**](../frames_rover/ackermann_rover.md): direction is controlled by pointing wheels in the direction of travel.
```

Comment:
Shouldn't we be "Ackermann" everywhere, not this mix of "ackermann"?

Reply:
Yes, I'll go through and make it consistent.

Reply:
Implemented in [9db2bef](https://github.com/PX4/PX4-user_guide/pull/3265/commits/9db2bef6c17098cb78f9137468e3970ad920071c)